lib/signs/bus.ex

```diff
@@ -841,4 +841,9 @@ defmodule Signs.Bus do
       )
     end
   end
+
+  # If a Silver Line sign is in headway mode, SignsUI will flag its predictions but RTS needs to treat it as off
+  defp config_off?("Silver_Line." <> _, :headway), do: true
```

Comment:
I don't have much of the context on RTS yet, but is there no more-robust way to tell whether a sign is part of the Silver Line group? (if "group" is even the right word) I'm wary of peeking into the contents of IDs.

Reply:
Unfortunately we don't have much: https://github.com/mbta/signs_ui/blob/f3c647b800a380649014f67eea97b1f9d6ff4abe/priv/signs.json#L8554. SL and bus share the same config shape so it's tough to tell them apart aside from the IDs. Maybe the best thing to do here is peak into `sources` and make sure that all `route_id`s are SL route IDs? I'm willing to bet an SL route ID change is a lot less likely than an ID overhaul so the code should be more stable.

Reply:
I think I found a way to make that work. We were already fetching all of the route IDs for alert purposes, so I just reused that value.